docs/tasks/debug-application-cluster/audit.md

```diff
@@ -111,9 +111,13 @@ When an event is processed, it's compared against the list of rules in order.
 The first matching rule sets the audit level of the event. The audit policy is
 defined by the [`audit.k8s.io` API group][audit-api].
 
+Audit policies now require kind and apiVersion fields to be parsed.
```

Comment:
I'd suggest we remove line 114, "now" is confusing. It is unclear to readers. They will ask ... "since when these two fields were required?" The fact is that the policy file has to have these two fields, though they were omitted in the doc.

Reply:
I have corrected it. plz check it ^_^